examples/blog/README.md

```diff
@@ -3,148 +3,312 @@
 ### Dispatch
 You will need a dispatch cluster deployed and configured, please follow the quick start instruction at [dispatch](https://github.com/vmware/dispatch) repository.
 
-You will need the url of your dispatch installation.
+For minikube deployment, keep a note of the port of your dispatch host and dispatch api-gateway host. This is printed
+out at the end of the dispatch installation, but you can easily fetch it via kubectl:
```

Comment:
I believe it is also in ~/.dispatch/config.json which might be easier to pull out than the port numbers in kubectl.

export DISPATCH_API_URL=http://$DISPATCH_HOST:$(jq '."api-https-port"' ~/.dispatch/config.json)

Reply:
This is going in the next checkin.